### PR TITLE
--http mode which only returns HTTP-related information

### PR DESCRIPTION
**Pull request by konklone**
This provides a tiny flag to only get the HTTP-related information, when there's no need to go make potentially expensive DNS calls.

### CODE REVIEW

File: lib/site-inspector.rb [hunk 1]
```diff
@@ -127,37 +127,50 @@ def to_json
...
+        :domain => domain.to_s,
+        :uri => uri.to_s,
+        :live => !!response,
+        :ssl => https?,
+        :enforce_https => enforce_https?,
+        :non_www => non_www?,
+        :redirect => redirect,
+        :headers => headers
+      }
+    else
```
Comment (benbalter) [inline lib/site-inspector.rb]: I'd have a basic hash (http_only), which then is concat'd with a larger hash if non-http-only, just to DRY things up a bit.

Comment (konklone) [inline lib/site-inspector.rb]: Well, we can just call this PR a wash if you want, since I replaced this approach with a much more wild one in #24.

**Comment by benbalter** [comment]
This can be closed in favor of #24, right?

**Comment by konklone** [comment]
Yep.

**Pull request closed without merging**